torch/utils/cpp_extension.py

```diff
@@ -1017,6 +1029,16 @@ def CUDAExtension(name, sources, *args, **kwargs):
     constructor. Full list arguments can be found at
     https://setuptools.pypa.io/en/latest/userguide/ext_modules.html#extension-api-reference
 
+    Note!
```

Comment:
Shouldn't this be `..note:` or something like that?

Reply:
I wasn't swayed either way, the cudaextension one had a big chunk of text so I stuck with it. but yea, can change